spec/lexer_spec.cr

```diff
@@ -140,7 +140,7 @@ describe Lexer do
   it_lexes_time "1979-05-27T07:32:00Z", Time.new(1979, 5, 27, 7, 32, 0, kind: Time::Kind::Utc)
   it_lexes_time "1979-05-27T07:32:00-07:30", Time.new(1979, 5, 27, 15, 2, 0, kind: Time::Kind::Utc)
   it_lexes_time "1979-05-27T07:32:00+07:30", Time.new(1979, 5, 27, 0, 2, 0, kind: Time::Kind::Utc)
-  it_lexes_time "1979-05-27T07:32:00.999999-07:00", Time.new(1979, 5, 27, 14, 32, 0, 999, kind: Time::Kind::Utc)
+  it_lexes_time "1979-05-27T07:32:00.999999-07:00", Time.new(1979, 5, 27, 14, 32, 0, nanosecond: 999, kind: Time::Kind::Utc)
```

Comment:
this spec should be `nanosecond: 999999000`. Otherwise the time would be "1979-05-27T07:32:00.000000999-07:00" (999 nanosecond only).

This will trigger that the changes in the lexer need to be `microseconds * 1000` since the argument was milliseconds in crystal 0.23